.github/workflows/azure-site-extension.yml

```diff
@@ -0,0 +1,80 @@
+name: Azure Site Extension
+
+on:
+  workflow_dispatch:
+
+env:
+  # Enable versioned runner quiet mode to make CI output easier to read:
+  # OUTPUT_MODE: quiet
+  SPEC_FILE_TEMPLATE: 'NewRelic.Azure.WebSites.Extension.NodeAgent.nuspec'
+
+jobs:
+  setup_tools:
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup dotnet '6.0.x'
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: '6.0.x'
+    - name: Display dotnet version
+      run: dotnet --version
+    - name: Display NuGet version
+      run: nuget ?
+
+  create_extension_bundle:
+    runs-on: windows-latest
+    needs:
+      - setup_tools
+
+    strategy:
+      matrix:
+        node-version: ['20.x'] # Node 22 isn't yet available on Azure
```

Comment:
```suggestion
        node-version: ['lts/*'] # Node 22 isn't yet available on Azure
```

Reply:
18.x and 20.x are available for Windows in Azure, so I was targeting those.